src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdCaseRetrieverTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.AUTH_DETAILS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.CASE_ID;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.CASE_REF;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.JURSIDICTION;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.SERVICE_TOKEN;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.THE_CASE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.USER_ID;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.USER_TOKEN;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.CcdCaseRetriever.CASE_TYPE_ID;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CcdCaseRetrieverTest {
+    @Mock
+    private CoreCaseDataApi dataApi;
+    @Mock
+    private CcdAuthService authenticator;
+
+    private CcdCaseRetriever retriever;
+
+    @Test
+    public void should_retrieve_case_successfully() {
```

Comment:
is there any benefit to this sort of test?

Reply:
yup ... the mocks are setup so that only if the correct information is presented will it actually pass the test. Its a wiring test.